### PR TITLE
fix: issue of translation for `TimeAgo.vue` 

### LINKED ISSUE #7862: [CW-2457] fix: Make activity timestamp messages translatable
These copies aren't translatable at the moment. 

ref: [https://github.com/chatwoot/chatwoot/blob/9322112481e6341b6b291d7d44b234434a2f6c9b/app/javascript/dashboard/components/ui/TimeAgo.vue#L56](https://github.com/chatwoot/chatwoot/blob/9322112481e6341b6b291d7d44b234434a2f6c9b/app/javascript/dashboard/components/ui/TimeAgo.vue#L56)

<sub>From [SyncLinear.com](https://synclinear.com) | [CW-2457](https://linear.app/chatwoot/issue/CW-2457/fix-make-activity-timestamp-messages-translatable)</sub>

### PR DESCRIPTION
**Pull request by jsobralgitpush**
# fix: issue of translation for `TimeAgo.vue`

## Description

This PR intents to solve issue #7862 , which refers to a lake of translations on `TimeAgo.vue` file. 
Fixes #7862 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Run the app locally
- Go to a single conversation
- Check the last activity 

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [] I have commented on my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by sojan-official** [comment]
Thank you so much for your contribution. We truly appreciate your efforts. 🙏 

Our team will prioritize reviewing and testing your changes according to our existing schedule and workload. 
Should there be a need for any further actions or information from your side, we'll follow up here. 📨

Once again, we're grateful for your involvement and contribution to our community. 🚀🌟 🙌

File: app/javascript/dashboard/i18n/locale/en/chatlist.json [hunk 1]
```diff
@@ -41,6 +41,16 @@
       "STATUS": "Status",
       "ORDER_BY": "Order by"
     },
+    "CHAT_TIME_STAMP": {
+      "CREATED": {
+        "LATEST": "Created",
+        "OLDEST": "Created at:"
+      },
+      "LAST_ACTIVITY": {
+        "NOT_ACTIVE": "Last activity :",
```
Comment (iamsivin) [inline app/javascript/dashboard/i18n/locale/en/chatlist.json:50]:
```suggestion
        "NOT_ACTIVE": "Last activity:",
```

File: app/javascript/dashboard/components/ui/TimeAgo.vue [hunk 2]
```diff
@@ -53,16 +53,16 @@ export default {
...
     },
     lastActivity() {
       const lastActivityTimeDiff =
         Date.now() - this.lastActivityTimestamp * 1000;
       const isNotActive = lastActivityTimeDiff > DAY_IN_MILLI_SECONDS * 30;
       return !isNotActive
-        ? `Last activity ${this.lastActivityAtTimeAgo}`
-        : `Last activity: ${this.dateFormat(this.lastActivityTimestamp)}`;
+        ? `${this.$t('CHAT_LIST.CHAT_TIME_STAMP.LAST_ACTIVITY.NOT_ACTIVE')} ${this.lastActivityAtTimeAgo}`
+        : `${this.$t('CHAT_LIST.CHAT_TIME_STAMP.LAST_ACTIVITY.ACTIVE')} ${this.dateFormat(this.lastActivityTimestamp)}`;
```
Comment (iamsivin) [inline app/javascript/dashboard/components/ui/TimeAgo.vue:65]:
```suggestion
        ? `${this.$t('CHAT_LIST.CHAT_TIME_STAMP.LAST_ACTIVITY.ACTIVE')} ${this.lastActivityAtTimeAgo}`
        : `${this.$t('CHAT_LIST.CHAT_TIME_STAMP.LAST_ACTIVITY.NOT_ACTIVE')} ${this.dateFormat(this.lastActivityTimestamp)}`;
```

File: app/javascript/dashboard/components/ui/TimeAgo.vue [hunk 3]
```diff
@@ -53,16 +53,16 @@ export default {
       const createdTimeDiff = Date.now() - this.createdAtTimestamp * 1000;
       const isBeforeAMonth = createdTimeDiff > DAY_IN_MILLI_SECONDS * 30;
       return !isBeforeAMonth
-        ? `Created ${this.createdAtTimeAgo}`
-        : `Created at: ${this.dateFormat(this.createdAtTimestamp)}`;
+        ? `${this.$t('CHAT_LIST.CHAT_TIME_STAMP.CREATED.LATEST')} ${this.createdAtTimeAgo}`
+        : `${this.$t('CUSTOM_ATTRIBUTES.VALIDATIONS.INVALID_URL')}: ${this.dateFormat(this.createdAtTimestamp)}`;
```
Comment (iamsivin) [inline app/javascript/dashboard/components/ui/TimeAgo.vue:57]:
```suggestion
        : `${this.$t('CHAT_LIST.CHAT_TIME_STAMP.CREATED.OLDEST')}: ${this.dateFormat(this.createdAtTimestamp)}`;
```

**Review by iamsivin — COMMENTED** [review]
@jsobralgitpush Thanks for the PR ❤️. Please see the inline comments and LMK if you need any help.

File: app/javascript/dashboard/components/ui/TimeAgo.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (iamsivin) [reply app/javascript/dashboard/components/ui/TimeAgo.vue:57]: image

File: app/javascript/dashboard/i18n/locale/en/chatlist.json (hunk 1 above)
Comment (jsobralgitpush) [reply app/javascript/dashboard/i18n/locale/en/chatlist.json]: Done!

File: app/javascript/dashboard/components/ui/TimeAgo.vue (hunk 2 above)
Comment (jsobralgitpush) [reply app/javascript/dashboard/components/ui/TimeAgo.vue:65]: Done!

File: app/javascript/dashboard/components/ui/TimeAgo.vue (hunk 3 above)
Comment (jsobralgitpush) [reply app/javascript/dashboard/components/ui/TimeAgo.vue]: Done!

[the remark this answers is not in the document]
Comment (jsobralgitpush) [reply app/javascript/dashboard/components/ui/TimeAgo.vue]: I think its ok now, @iamsivin !

**Review by iamsivin — APPROVED** [review]
Thanks for the PR🎉. I tested locally, and LGTM.

**Pull request merged**